### ISSUE TITLE
Shotwell app flathub version is unresponsive, freezing and crashing on Silverblue.

### ISSUE THREAD

**Issue by queeup** [fedora-silverblue/issue-tracker#357]
**Describe the bug**
Shotwell app is freezing and crashing after [#32](https://github.com/flathub/org.gnome.Shotwell/pull/32) & [#33](https://github.com/flathub/org.gnome.Shotwell/pull/33) update.

**To Reproduce**
1. Add flathub repo
2. İnstall flatub Shotwell version
3. Open Shotwell app

**Expected behavior**
No freeze or crash.

**Screenshots**
N/A

**OS version:**
```bash
rpm-ostree status -b
State: idle
AutomaticUpdates: stage; rpm-ostreed-automatic.timer: inactive
BootedDeployment:
● fedora:fedora/37/x86_64/silverblue
                  Version: 37.20220923.n.0 (2022-09-23T07:57:29Z)
               BaseCommit: 58ce64d1a3405b5d72d0840da30e7bc88d341f17addfe7a00bbd269c09623b73
             GPGSignature: Valid signature by ACB5EE4E831C74BB7C168D27F55AD3FB5323552A
      RemovedBasePackages: firefox firefox-langpacks 104.0.2-1.fc37 gnome-tour 43.0-1.fc37 toolbox 0.0.99.3-7.fc37
          LayeredPackages: code ffmpegthumbnailer langpacks-tr pipenv printer-driver-brlaser rpmfusion-free-release
                           rpmfusion-nonfree-release tailscale thinkfan

AvailableUpdate:
           Diff: 1 upgraded

```
**Additional context**
Shotwell app on fedora flatpak repo and Workstation versions are working fine.

Upstream bug report: https://github.com/flathub/org.gnome.Shotwell/issues/39

_Note:_ All bug reports are in the upstream bug report.

**Comment by travier** [comment]
As you've already reported it upstream to Flathub, there is nothing to do in Silverblue so I'll close this one.

**Comment by queeup** [comment]
Yes but there is something wrong with silverblue on this. Thats why I opened it here. We couldn't figure it out. Shotwell app working well in other fedoras except silverblue.

**Comment by travier** [comment]
Is this still an issue?

**Comment by queeup** [comment]
Yes it is. With silverblue 38 and flathub shotwell I have same issue. Only fedora flatpak repo shotwell working.

**Comment by queeup** [comment]
Connected issues:

https://gitlab.gnome.org/GNOME/shotwell/-/issues/5002
https://github.com/gphoto/libgphoto2/issues/847

**Comment by travier** [comment]
Is this still an issue on Fedora Silverblue 39? Thanks

**Comment by queeup** [comment]
Since then I am using fedora flatpak version. I will test and report.

**Comment by queeup** [comment]
Problem still exists with flathub version of shotwell.

Flathub version: 0.32.3
